cli/regex.py

```diff
@@ -1,5 +1,4 @@
 RG_NAME_PATTERN = '^[-\w\._\(\)]{1,90}[^\.]$'
-LOCATION_PATTERN = '^[a-zA-Z]+2?$'
 STORAGE_NAME_PATTERN = '^[a-z0-9]{3,24}$'
```

Comment:
Same for here as comment on line 1.

Reply:
I actually removed the `LOCATION_PATTERN` so I don't need to document it here, but I'll definitely document the other regexes in a separate PR.